app/src/main/java/com/globant/utils/Data.kt

```diff
@@ -5,4 +5,14 @@ package com.globant.utils
  */
 data class Data<RequestData>(var responseType: Status, var data: RequestData? = null, var error: Exception? = null)
 
-enum class Status { SUCCESSFUL, ERROR, LOADING, GetCharacterSuccess, GetCharacterError, GetCharacterByIdSuccess, GetCharacterByIdError }
+enum class Status {
+    SUCCESSFUL,
+    ERROR,
+    LOADING,
+    GetCharacterSuccess,
```

Comment:
as this is an enum all variables should be in uppercase and the words separated by _

Reply:
done, thanks mau!